graphql/server.go

```diff
@@ -0,0 +1,28 @@
+package graphql
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/99designs/gqlgen/graphql/handler"
+	"github.com/99designs/gqlgen/graphql/playground"
+)
+
+const defaultPort = "8080"
+
+// StartServer starts the graphql server
+func StartServer() {
+	port := os.Getenv("PORT")
```

Comment:
I think this is probably moot in light of the next comment in this file, but in general we should not derive "magic state" in functions that take no arguments. In situations like this, ports should be passed in as arguments, and also functions should return errors if there's a problem